### DISCUSSION TITLE
select with encode,

### DISCUSSION THREAD

**Discussion by XRFate** [supabase/supabase#29408] [Questions]
Is it possible to select with encode like SQL does while uses supabase.js
```
 .select(`encode(data::bytea, 'hex') as data`) 

```
[similar discusstion](https://github.com/orgs/supabase/discussions/2441)

**Comment by silentworks** [comment]
No this is not possible with the normal `.select` call. You can however do this in a postgres function and call it via the `.rpc` call.
